src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -379,6 +399,87 @@ def is_guid(guid):
         return False
 
 
+def get_latest_extension_version(extension_name='connectedk8s'):
+    try:
+        import re
+        git_url = "https://raw.githubusercontent.com/Azure/azure-cli-extensions/master/src/{}/setup.py".format(extension_name)
```

Comment:
Version present in seup.py might not always be the latest released version. Latest released version is always found in index.json: https://raw.githubusercontent.com/Azure/azure-cli-extensions/main/src/index.json
Alternatively, you could also do what this cmd does: 'az extension list-versions -n connectedk8s'

Reply:
Yes, this I also had in mind, Earlier Feng had told to use setup.py itself, but later I realized, the latest is there in index.json. Will update.